package.json

```diff
@@ -1,22 +1,25 @@
 {
-  "name": "brunch-app",
-  "description": "Description",
-  "author": "Your Name",
-  "version": "0.1.0",
-  "repository": {
-    "type": "git",
-    "url": ""
-  },
-  "scripts": {
-    "start": "brunch watch --server",
-    "build": "brunch build --production"
-  },
-  "dependencies": {},
-  "devDependencies": {
-    "javascript-brunch": "^2.0.0",
-    "css-brunch": "^2.0.0",
-    "uglify-js-brunch": "^2.0.0",
-    "clean-css-brunch": "^2.0.0",
-    "auto-reload-brunch": "^2.0.0"
-  }
+	"name": "brunch-app",
+	"description": "Description",
+	"author": "Your Name",
+	"version": "0.1.0",
+	"repository": {
+		"type": "git",
+		"url": ""
+	},
+	"scripts": {
+		"start": "brunch watch --server",
+		"build": "brunch build --production",
+		"postinstall": "node skeleton.js"
+	},
+	"dependencies": {
+		"website-scraper": "^0.3.3"
```

Comment:
Can we move website-scraper to devDependencies, since so far it seems like its a Brunch convention to leave the main dependencies config for target project dependencies, like for example Natalia could put d3 and other frontend libs in this section. All skeleton related deps can live under devDependencies to prevent confusion as to what modules are for what purpose.